packages/components/app/styles/components/modal.scss

```diff
@@ -0,0 +1,156 @@
+//
+// MODAL
+//
+
+@use "../mixins/focus-ring" as *;
+
+.hds-modal {
+  flex-direction: column;
+  padding: 0;
+  background: var(--token-color-surface-primary);
+  border: none;
+  border-radius: 10px;
+  box-shadow: var(--token-surface-overlay-box-shadow);
+
+  &[open] {
+    position: fixed;
+    display: flex;
+  }
+
+  &::backdrop {
+    display: none;
+  }
+}
+
+.hds-modal__overlay {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background: var(--token-color-palette-neutral-700);
+  opacity: 0.5;
+}
+
+.hds-modal__header {
+  display: flex;
+  flex: none;
+  align-items: flex-start;
+  padding: 16px 24px;
+  border-top-left-radius: inherit;
+  border-top-right-radius: inherit;
+}
+
+.hds-modal__icon {
+  flex: none;
+  align-self: center;
+  margin-right: 16px;
+}
+
+.hds-modal__title {
+  flex-grow: 1;
+}
+
+.hds-modal__tagline {
+  margin-bottom: 4px;
+}
+
+.hds-modal__dismiss {
+  flex: none;
+  align-self: center;
+  margin-left: 16px;
+  padding: 4px;
```

Comment:
could it be that this padding makes it visually different than the similar dismiss in the alert? plus, is not in the same position (16px from the right side for the alert, 24px for the dismiss)
maybe something @jorytindall can confirm?

Reply:
yes, the extra padding made it bigger, but now that I abstracted the `DismissButton` they look the same across components. in terms of 'positioning', the header padding is indeed `24px` at the moment (similarly with the other modal containers: body and footer) – would be good to get confirmation from Jory that this is how was intended